### PR TITLE
Manually registered IWantToRuns should not be scanned

### PR DESCRIPTION
**Pull request by colin-higgins**
Connects to https://github.com/Particular/NServiceBus.Host/issues/66

### CODE REVIEW

**Comment by colin-higgins** [comment]
ping @Particular/nservicebus-maintainers 

If this gets accepted, I can include it in Azure Host as well and close that out.

**Comment by andreasohlund** [comment]
Looks good, @bording can you give this a once over and pull?

File: src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableFeature.cs [hunk 1]
```diff
@@ -16,12 +16,15 @@ public StartableAndStoppableFeature()
         protected override void Setup(FeatureConfigurationContext context)
         {
             var startableStoppableInstances =
-                context.Settings.GetOrDefault<List<IWantToRunWhenEndpointStartsAndStops>>()
-                ?? new List<IWantToRunWhenEndpointStartsAndStops>();
+                            context.Settings.GetOrDefault<List<IWantToRunWhenEndpointStartsAndStops>>()
+                            ?? new List<IWantToRunWhenEndpointStartsAndStops>();
```
Comment (bording) [inline src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableFeature.cs]: Minor nitpick, but this looks like it's indented too far.

Comment (colin-higgins) [inline src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableFeature.cs]: Ah, copy paste problems :)

**Pull request merged**